pkg/distributor/push.go

```diff
@@ -306,16 +305,16 @@ func calculateRetryAfter(retryAttemptHeader string, baseSeconds int, maxBackoffE
 // toGRPCHTTPStatus converts the given error into an appropriate GRPC and HTTP status corresponding
 // to that error, if the error is one of the errors from this package. Otherwise, codes.Internal and
 // http.StatusInternalServerError is returned.
-func toGRPCHTTPStatus(ctx context.Context, pushErr error, limits *validation.Overrides) (codes.Code, int) {
+func toGRPCHTTPStatus(ctx context.Context, pushErr error, limits *validation.Overrides) int {
```

Comment:
@ying-jeanne thank you for addressing this request. 
The very last thing is to rename it back to `toHTTPStatus()` since it actually returns an HTTP status. There is no gRPC code returned by this function.
Please update the test name too.

Reply:
it is already updated in the next commit